tst/standard/labels.tst

```diff
@@ -79,6 +79,14 @@ gap> DigraphVertexLabels(gr);
 [ 1, 3, 4, 5, 6, 7, 8, 9, 10 ]
 gap> D := NullDigraph(5);;
 gap> RemoveDigraphVertexLabel(D, 6);
+gap> A := Immutable([1, 2]);
+[ 1, 2 ]
+gap> D := CycleDigraph(2);
+<immutable cycle digraph with 2 vertices>
+gap> SetDigraphVertexLabels(D, A);
+gap> SetDigraphVertexLabels(D, ["a", "b"]);
+gap> DigraphVertexLabels(D);
+[ "a", "b" ]
```

Comment:
This does not capture the previous 'bad' behaviour. Indeed, in the current `stable-1.4` branch, this already works totally fine:
```
gap> A := Immutable([1, 2]);
[ 1, 2 ]
gap> D := CycleDigraph(2);
<immutable cycle digraph with 2 vertices>
gap> SetDigraphVertexLabels(D, A);
gap> SetDigraphVertexLabels(D, ["a", "b"]);
gap> DigraphVertexLabels(D);
[ "a", "b" ]
```
Rather, the problem was that the immutable list of vertex labels meant that you couldn't change _individual_ vertex labels (or proper subsets of labels):
```
gap> A := Immutable([1, 2]);;
gap> DigraphVertexLabels(D);;
gap> SetDigraphVertexLabels(D, A);
gap> SetDigraphVertexLabel(D, 2, "b");
Error, List Assignment: <list> must be a mutable list in
  D!.vertexlabels[v] := name; at /Users/Wilf/gap/pkg/digraphs/gap/labels.gi:35 called from
<function "SetDigraphVertexLabel for a digraph, pos int, object">( <arguments>\
 )
 called from read-eval loop at *stdin*:14
type 'quit;' to quit to outer loop
brk>
```
So could you please update the test to reflect this use case, i.e. the test should be:
```
gap> A := Immutable([1, 2]);;
gap> D := CycleDigraph(2);
gap> SetDigraphVertexLabels(D, A);
gap> IsMutable(DigraphVertexLabels(D));
true
gap> SetDigraphVertexLabel(D, 2, "b");
gap> DigraphVertexLabels(D);
[ 1, "b" ]
```

Reply:
Ah yes, sorry! It actually appears I made a typo in my discussion on the issue: I'd written `gap> SetDigraphVertexLabel(D, ["a", "b"]);`, and clearly this didn't work because the function should've been `SetDigraphVertexLabels` instead, and I just copied that over from the issue to the tests but with the correct function. I'll fix that now!